README.adoc

```diff
@@ -93,3 +93,19 @@ make sure the installer actually downloads and installs the new DFX release the
    (the CD script) and execute it.
 
 
+== Troubleshooting
+This section provides solutions to problems you might encounter when using `dfx`
+
+=== Project Reset
+
+This command will remove the build directory and restart your replica
+```shell
+dfx stop && dfx start --clean --background
+```
```

Comment:
Since this is an asciidoc file, not markdown, the formatting is a little different:
```suggestion
[source, bash]
----
dfx stop && dfx start --clean --background
----
```